teams/alumni.toml

```diff
@@ -1,36 +1,22 @@
-# DEPRECATED: Do not add new members to this team, instead add to `people.alumni`
-# in the team's file or create a new `people.alumni` key and add the
-# person's name there instead.
 name = "alumni"
 
 [people]
 leads = []
+# Contributors whose former teams no longer correspond to any page on the
```

Comment:
Which teams no longer exist? We have the ability to archive a team. Perhaps the right approach is to bring those teams back in archived form?

Reply:
Docs team, style team, and community team are a few that come to mind

Reply:
I have updated my script in #1125 to populate archived teams, too.

Among the contributors already pinged in this PR, this puts QuietMisdreavus and frewsxcv on the archived docs team (as they've already pointed out!) and QuietMisdreavus on an archived devtools-peers team (see https://github.com/rust-lang/prev.rust-lang.org/commit/2b61fac83c59ba8e1d911e6760ad07883bea6aee &ndash; https://github.com/rust-lang/prev.rust-lang.org/commit/3042e24daa5d1bddf7b1396c5ad2ac3395b3fccb).

I'll make a separate PR to fill in archived teams for the remaining contributors whom this PR left in alumni.toml.